src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,105 @@
+import { useEffect, useState } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { ErrorMessage } from '../components/ErrorMessage';
+import { NoPeopleMessage } from '../components/NoPeopleMessage';
+import { PeopleTable } from '../components/PeopleTable';
+
+import { useSort } from '../hooks/useSort';
+import { getSearchWith } from '../utils/searchHelper';
+import { getPeople } from '../api';
+
+import { Gender } from '../types/SortSex';
+import { Sort } from '../types/Sort';
+import { Person } from '../types/Person';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const { personSlug = '' } = useParams();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || Gender.All;
+  const query = searchParams.get('query') || '';
+  const sort = searchParams.get('sort') || Sort.None;
+  const order = searchParams.get('order') || '';
+
```

Comment:
move all these search param names to some enum

Reply:
done